lib/Control/IO/Region/Internal.hs

```diff
@@ -5,7 +5,7 @@
 module Control.IO.Region.Internal
 where
 
-import Prelude (($!), Enum(..))
+--import Prelude (($!), Enum(..))
```

Comment:
GHC has `[]` available without any import (?) while MicroHs exports it from the Prelude